sunpy/physics/transforms/differential_rotation.py

```diff
@@ -1,15 +1,20 @@
 from __future__ import division
 
-__all__ = ['diff_rot']
 import numpy as np
 import datetime
 from astropy import units as u
-from astropy.coordinates import Longitude
+from astropy.coordinates import Longitude, Latitude, Angle
+from sunpy.time import parse_time, julian_day
```

Comment:
the linebreak in these imports should seperate the external imports from the sunpy ones.